eslintrc.js

```diff
@@ -0,0 +1,199 @@
+module.exports = {
+    env: {
+        node: true,
+    },
+    parser: "@typescript-eslint/parser",
+    parserOptions: {
+        project: "tsconfig.json",
+        sourceType: "module",
+    },
+    plugins: ["@typescript-eslint", "@typescript-eslint/tslint"],
+    rules: {
```

Comment:
This is a lot of rules to enable! Let's extend from the list of rules defined in the plugins.

Example reference: https://github.com/typescript-eslint/tslint-to-eslint-config/pull/229/files#diff-e4403a877d80de653400d88d85e4801aR6